scripts/cgm_adapter.py

```diff
@@ -279,30 +304,38 @@ def _run(self):
         self.validation_exceptions.clear()
         log.info(f'Version set to: {self.timestamp}')
         projects = self.parse_csv()
+        catalog = self.args.catalog
         completed, skipped = 0, 0
         for project in projects.values():
-            project_id = project['project_uuid']
-            if self.args.catalog and not self.project_in_catalog(project_id):
+            if catalog and not self.project_in_catalog(project, catalog):
                 skipped += 1
-                log.info(f'Skipped project {project_id!r} not found in {self.args.catalog!r}')
+                project_id = project['project_uuid']
+                log.info(f'Skipped project {project_id!r} not found in {catalog!r}')
             else:
                 result = self.process_project(project)
                 completed += 1 if result else 0
         log.info(f'Completed staging {completed} projects.')
         if skipped:
-            log.info(f'Skipped {skipped} projects not found in {self.args.catalog!r}')
-
-    def project_in_catalog(self, project_id: str) -> bool:
-        url = furl('https://service.dev.singlecell.gi.ucsc.edu',
-                   path=f'/index/projects/{project_id}',
-                   args=dict(catalog=self.args.catalog)).url
-        response = requests.get(url)
-        if response.status_code == 200:
-            return True
-        elif response.status_code == 404:
-            return False
+            log.info(f'Skipped {skipped} projects not found in {catalog!r}')
+
+    def project_in_catalog(self, project: Mapping[str, Any], catalog: str) -> bool:
+        # If project's catalog was given on spreadsheet, check against that
+        project_catalog = project['catalog']
+        if project_catalog:
+            return project_catalog == catalog
+        # Otherwise ping Azul to check if the project exists in the catalog.
         else:
-            raise Exception(f'Unexpected response status {response.status_code!r} from {url!r}')
+            project_id = project['project_uuid']
+            url = furl('https://service.azul.data.humancellatlas.org/',
+                       path=f'/index/projects/{project_id}',
```

Comment:
```suggestion
                       path=('index', 'projects', project_id),
```
`furl.path` allows for usage of sequences, that way we don't have to use f-strings to concat.